### ISSUE TITLE
❗N67 support timeline

### ISSUE THREAD

**Issue by AndrewAnnex** [AndrewAnnex/SpiceyPy#433]
the NAIF just released N67 spice which includes a number of new functions and enhancements https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/info/whatsnew.html. 

Unfortunately, the release coincides with a very busy period in my life, so realistically I won't be able to get started on updating spiceypy to N67 until mid February 2022.

User contributions to add N67 support are welcome, but PR merges are paused until mid February also.

**Comment by AndrewAnnex** [comment]
As an update I just merged #439 which is allowing spiceypy to build N67 cspice, although no new wrapper functions are present this allows me to move forward with a new release. I may need to think about this as it may not be a major breaking change, but in the past I've bumped major versions just for new cspice releases. Likely that makes more sense as adding the wrapper functions are not breaking changes, but do cause the api to advance.


road map is:

1 [x] bump to n67

2 [] major release to allow conda forge to move forward

3 [] minor/patch releases for new wrapper functions as they arrive
